service/src/main/java/uk/nhs/adaptors/gp2gp/common/service/XPathService.java

```diff
@@ -0,0 +1,47 @@
+package uk.nhs.adaptors.gp2gp.common.service;
+
+import java.io.IOException;
+import java.io.StringReader;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathConstants;
+import javax.xml.xpath.XPathExpression;
+import javax.xml.xpath.XPathExpressionException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.w3c.dom.Document;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+@Service
+public class XPathService {
+    @Autowired
+    private XPath xPath;
```

Comment:
Don't inject those built-ins. We don't need any additional control over it and we won't be replacing it. It just adds unnecessary complication